src/components/dashboard/DashboardView.js

```diff
@@ -33,6 +34,9 @@ const DashboardView = ({user, type, linkHandler, activeComponent, headerLinks, l
     case "My Jobs":
       currentComponent = (<MyJobOffers user={user} loading={loading}/>);
       break;
+    case "My Profile":
```

Comment:
Please integrate everything into the existing profile and remove the lines you added to the Dashboard, DashboardView etc.

Reply:
Done!